crates/bevy_ecs/src/schedule/set.rs

```diff
@@ -175,6 +175,18 @@ define_label!(
     }
 );
 
+impl dyn SystemSet {
```

Comment:
Another option here is to remove these methods entirely by inlining their definitions.  `debug_name` is only used in one place, and `is_system_type()` in two, and writing `system_type().is_some()` in those places isn't really any longer or less clear.